maskrcnn_benchmark/engine/inference.py

```diff
@@ -6,16 +6,18 @@
 import os
 from collections import OrderedDict
 
-import torch
+import numpy as np
 
+from ..config import cfg
+import torch
 from tqdm import tqdm
-
+from ..data.datasets.voc import VOC_BBOX_LABEL_NAMES
```

Comment:
Ideally we would keep the inference engine agnostic to particular dataset implementations. But this will require some refactorings.

Reply:
I removed it.